.changeset/friendly-bananas-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+'next-tinacms-cloudinary': minor
+---
+
+The `persist` method on the Cloudinary Media Store was not returning a properly parsed response from the Cloudinary API. As such, the ImageFieldPlugin could not properly set the image field on drag and drop. The latest changes add new formatting logic to `persist`.
```

Comment:
```suggestion
Fix drag + drop behavior in ImageFieldPlugin
```

Here's my suggestion for the changelog entry. They don't always have to be this terse but IMO it should focus on what the impact of updating the package will be for the developer using it, and doesn't need to go deep into internal details